README.md

```diff
@@ -93,6 +93,10 @@ It is based upon and contains a lot of code written by [Jeroen De Dauw]
 
 ## Release notes
 
+### 1.1.7 (2016-05-25)
+
+* Fixed `FORMAT_NAME` string constants in all parsers
```

Comment:
Ok, but what should the release notes say then? I would like to do a patch-level or minor release, but what from https://github.com/DataValues/Geo/compare/1.1.6...master is worth mentioning?